databricks/koalas/frame.py

```diff
@@ -2626,7 +2657,14 @@ def describe(self) -> 'DataFrame':
         if len(exprs) == 0:
             raise ValueError("Cannot describe a DataFrame without columns")
 
-        sdf = self._sdf.select(*exprs).describe()
+        if any((p <= 0.0) | (p >= 1.0) for p in percentiles):
+            raise ValueError("Percentiles not in range (0.0, 1.0)")
+
+        formatted_perc = ["{:.0%}".format(p) for p in percentiles]
```

Comment:
Seems like pandas adds `50%` if the `percentiles` doesn't include the value, i.e., the example `df.describe(percentiles = [0.15, 0.85])` above includes `50%` in the result.

```py
>>> df = pd.DataFrame({'numeric1': [1, 2, 3], 'numeric2': [4.0, 5.0, 6.0]})
>>> df.describe(percentiles=[0.15, 0.85])
       numeric1  numeric2
count       3.0       3.0
mean        2.0       5.0
std         1.0       1.0
min         1.0       4.0
15%         1.3       4.3
50%         2.0       5.0
85%         2.7       5.7
max         3.0       6.0
```

Could you follow the behavior?

Reply:
Done in the new version + I'm sorting the percentiles (following the pandas behavior)